dag.yml

```diff
@@ -71,30 +71,31 @@ steps:
   data://meadow/oecd/2016-06-01/regional_wellbeing:
     - walden://oecd/2016-06-01/regional_wellbeing
   data://garden/sdg/latest/sdg:
+    - backport://backport/owid/latest/dataset_1047_world_bank_gender_statistics__gender
```

Comment:
Not a bad idea, I'm not gonna do it now, but keep it in the back of my mind (it's not as long to deserve a separate file and it might go away soon anyway).